app/views/contexts/thank_you_context.py

```diff
@@ -45,11 +45,22 @@ def build_thank_you_context(
 
 
 def build_view_submitted_response_context(schema, submitted_at):
+
     view_submitted_response = {"enabled": schema.is_view_submitted_response_enabled}
 
     if schema.is_view_submitted_response_enabled:
+        view_submitted_expiry_seconds = current_app.config[
+            "VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS"
+        ]
+        response_expires_at = submitted_at + timedelta(
+            seconds=view_submitted_expiry_seconds
+        )
         expired = has_view_submitted_response_expired(submitted_at)
```

Comment:
We still need `VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS` for calculation of `nojsText` minutes , don't see reason to have another function if it can be done within the function

Reply:
i used your suggestion as the `nojsText` is a hardcoded 45 minutes text